### PR TITLE
Overhauled Docs

### PR DESCRIPTION
**Pull request by child-duckling**
- Added docs/ structure and created a visual for installing the pwa
- fixed typo  (#165)
- moved FAQ and local development to docs/faq and docs/local-dev respectively.

### CODE REVIEW

**Comment by child-duckling** [comment]
- Linked README to /docs files
- created redundancy in documentation so that people can't say "where's the docs?"
- created Wiki tab and c&p'ed everything from the README and /docs to the wiki
- created donation sidebar in wiki

**Comment by child-duckling** [comment]
- fixed image link in wiki/faq

**Comment by child-duckling** [comment]
- added #156

File: README.md [hunk 1]
```diff
@@ -10,8 +11,10 @@
 * [NodeJS](https://nodejs.org/en/)
 * [Material Design](https://material.google.com/)
 
+> If you want to self-host your own instance, [you can with docker.](/docs/local-dev.md)
+
 ## Support the Snapdrop Community
-Snadprop is free. Still, we have to pay for the server. If you want to contribute, please use PayPal:
+Snapdrop is free. Still, we have to pay for the server. If you want to contribute, please use PayPal:
```
Comment (child-duckling) [inline README.md:17]: #165

**Comment by RobinLinus** [comment]
#166 fixes this

**Pull request merged**